.github/workflows/java-publish-sonatype.yml

```diff
@@ -8,6 +8,17 @@ on:
     - gradle.properties
 
 jobs:
+  api-compatibility-check-core:
+    uses: th2-net/.github/.github/workflows/japi-compliance-checker.yml@main
+    with:
+      projectPath: ./cradle-core
+      targetJarUrl: https://s01.oss.sonatype.org/content/repositories/releases/com/exactpro/th2/cradle-core/5.0.2-dev/cradle-core-5.0.2-dev.jar
+  api-compatibility-check-cassandra:
+    uses: th2-net/.github/.github/workflows/japi-compliance-checker.yml@main
+    with:
+      projectPath: ./cradle-cassandra
+      continueOnError: true
+      targetJarUrl: https://s01.oss.sonatype.org/content/repositories/releases/com/exactpro/th2/cradle-cassandra/5.0.2-dev/cradle-cassandra-5.0.2-dev.jar
```

Comment:
I think we should use the latest non-dev release here

Reply:
`We need to check if we can configure the verification to ignore those classes` -> we can just remove cradle-cassandra check from workflow and only check cradle core API. I included both of them just to show how it works.

Reply:
If we use latest non-dev release we probably will have issues in public API, as I remember we did have some breaking changes. That's why I decided to use lates compatible version, so we can control compatibility starting from it